pulp_container/app/replica.py

```diff
@@ -0,0 +1,56 @@
+from pulpcore.plugin.replica import Replicator
+from pulpcore.plugin.util import get_url
+
+from pulp_glue.container.context import (
+    PulpContainerDistributionContext,
+    PulpContainerRepositoryContext,
+)
+
+from pulp_container.app.models import ContainerDistribution, ContainerRemote, ContainerRepository
+from pulp_container.app.tasks import synchronize as container_synchronize
+
+
+class ContainerReplicator(Replicator):
+    distribution_ctx_cls = PulpContainerDistributionContext
```

Comment:
pulp container plugin contains 2 distribution types(regular and pull-thru), if we replicate one pulp to another then we will create only regular distribution type and ignore pull-thru cache ones, which is probably fine(?) because we will mirror its 'sub-distributions', thoughts?
As a result of this replication process, the replicated instance will contain only regular repos, remotes and distributions and it will work fine, just it will not be a complete replication of original pulp and definitely these 2 instances will not be interchangeable